.github/workflows/r-test.yml

```diff
@@ -0,0 +1,74 @@
+name: testthat
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
```

Comment:
I am ok with this, but does it actually make sense to "merge" this github action with the other R-CMD-Check so that "deployment" doesn't happen without testing.

Reply:
We do want to always run the tests before deployment. I don't really care if they run as part of the same action, or a different action, as long as they always get run.

Reply:
I'll resolve this for now in that case and keep them as separate actions